lib/tasks/answers.rake

```diff
@@ -0,0 +1,30 @@
+namespace :questionable do
+  def error_message(answer)
+    puts "Answer #{answer.id} could not be updated"
+  end
+
+  desc 'Fills in all question_id columns on answers'
+  task update_answers_question_id: :environment do
+    updated = 0
+    count = Questionable::Answer.count
+    
+    Questionable::Answer.find_in_batches(batch_size: 100) do |answers|
```

Comment:
This should probably do `Questionable::Answer.where(question_id: nil).includes(:question)` instead to both not iterate over records that have already been filled in and also to fix N+1 on `answer.question`.